website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,83 @@
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/job-commands.gif" title="Configuring checkbox and commands list"/>
+
+
+### Built-in commands
+
+Every job invocation automatically includes the [`dbt deps`](/reference/commands/deps) command, meaning you don't need to add it to the **Commands** list in your job settings.  You will also notice every job will include a run step to reclone your repository and connect to your data platform, which can affect your job status if these run steps aren't successful.
+
+**Job outcome** &mdash; During a job run, the built-in commands are "chained" together.  This means if one of the run steps in the chain fails, then the next commands aren't executed, and the entire job fails with an "Error" job status.
+
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/fail-dbtdeps.jpg" title="A failed job that had an error during the dbt deps run step."/>
+
+### Checkbox commands
+
+For every job, you have the option to select the [Generate docs on run](/docs/collaborate/build-and-view-your-docs) or [Run source freshness](/docs/deploy/source-freshness) checkboxes. These checkboxes enable you to generate project docs or enable source freshness automatically.
+
+**Job outcome Generate docs on run checkbox** &mdash; dbt Cloud executes the checkbox, which uses the `dbt docs generate` command, _after_ the listed commands. If that particular run step in your job fails, the job can still succeed if all subsequent run steps are successful. Read [Build and view your docs](/docs/collaborate/build-and-view-your-docs) for more info.
+
+**Job outcome Source freshness checkbox** &mdash; dbt Cloud executes the checkbox, which uses the `dbt source freshness` command, as the first run step in your job. If that particular run step in your jobs fails, the job can still succeed if all subsequent run steps are successful. Read [Source freshness](/docs/deploy/source-freshness) for more info.
+
+### Command list
+
+You can add or remove as many [dbt commands](/reference/dbt-commands) as necessary for every job, however, you need to have at least one dbt command. 
+Commands labeled as "CLI only" in the [dbt Command reference doc](/reference/dbt-commands) are meant for [CLI](/docs/get-started/about-the-cli)-use only. In dbt Cloud, you can only run commands that _aren't labeled_ "CLI only".
+
+
+:::tip Using selectors
+
+Use [selectors](/reference/node-selection/syntax) as a powerful way to select and execute portions of your project in a job run. If a selector doesn't match any models, the job run won't consider it a failure. For example, to run tests for one_specific_model, use the selector: `dbt test --select one_specific_model`
+
+:::
+   
+    
+**Job outcome**  &mdash;  During a job run, the commands are “chained” together. This means if one of the run steps in the chain fails, then the next ones aren't executed. For example, if the 5th run step fails:
```

Comment:
```suggestion
**Job outcome**  &mdash;  During a job run, the commands are “chained” together. If one of the run steps in the chain fails, then the following steps aren't executed, and the job will fail. For example, if the 5th run step fails:
```

Reply:
Is there supposed to be an example in here where this is the fifth step?

Reply:
yes it's supposed to be tied to the screenshot but i realized it's not explicit. new wording now!

Reply:
ok reworded:

>In the following example image, the first four run steps are successful. However, if the fifth run step (`dbt run --select state:modified+ --full-refresh --fail-fast`) fails, then the next run steps aren't executed, and the entire job fails. The failed job returns a non-zero [exit code](/reference/exit-codes) and "Error" job status: